src/data/move-attrs/present-power-attr.ts

```diff
@@ -19,16 +22,16 @@ export class PresentPowerAttr extends VariablePowerAttr {
      * If this move is multi-hit, and this attribute is applied to any hit
      * other than the first, this move cannot result in a heal.
      */
-    const firstHit = user.turnData.hitCount === user.turnData.hitsLeft;
+    const isFirstHit = user.turnData.hitCount === user.turnData.hitsLeft;
```

Comment:
```suggestion
    const isFirstHit = user.turnData.hitCount === user.turnData.hitsLeft && user.turnData.hitsLeft > 0;
```
(An extra safeguard to prevent simulated calls from healing the target)